### PR TITLE
Add settle command to UG

### LINKED ISSUE #128: Edit UG to include settle command

### PR DESCRIPTION
**Pull request by Soumyadip-cmd**
Closes #128

### CODE REVIEW

File: docs/UserGuide.md
Comment (Feng1231) [inline docs/UserGuide.md:1]: LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -215,3 +233,4 @@ Action     | Format, Examples
 **Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List**   | `list`
 **Help**   | `help`
+**Settle** | `settle INDEX amount/AMOUNT`
```
Comment (TY1Fan) [inline docs/UserGuide.md:236]: I think you should in an e.g like the find command above

**Review by teamotitan — APPROVED** [review]
LGTM

**Review by Feng1231 — COMMENTED** [review]
lgtm

**Pull request merged**